src/main/java/org/terasology/metalrenegades/economy/actions/ShowDialogActionTypeHandler.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2019 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.terasology.metalrenegades.economy.actions;
+
+import com.google.common.collect.ImmutableMap;
+import org.terasology.persistence.typeHandling.DeserializationContext;
+import org.terasology.persistence.typeHandling.PersistedData;
+import org.terasology.persistence.typeHandling.PersistedDataMap;
+import org.terasology.persistence.typeHandling.RegisterTypeHandler;
+import org.terasology.persistence.typeHandling.SerializationContext;
+import org.terasology.persistence.typeHandling.SimpleTypeHandler;
+
+import java.util.Map;
+
+@RegisterTypeHandler
+public class ShowDialogActionTypeHandler extends SimpleTypeHandler<ShowDialogAction> {
+
+    @Override
+    public PersistedData serialize(ShowDialogAction action, SerializationContext context) {
```

Comment:
Did you check if this still works with the new serialization that is about to be merged soon in the main repository?

Reply:
Yes, it works with the new system. Just need to delete these two files. I've mentioned this is #23 for now.